pylspclient/lsp_structs.py

```diff
@@ -26,6 +26,47 @@ def __init__(self, line, character, offset=0):
         self.character = character
         self.offset = offset
 
+    def __repr__(self) -> str:
+        return str(
+            {"line": self.line, "character": self.character, "offset": self.offset}
+        )
+
+    def __eq__(self, __value: object) -> bool:
+        return isinstance(__value, Position) and (self.line, self.character) == (
+            __value.line,
+            __value.character,
+        )
+
+    def __ne__(self, __value: object) -> bool:
+        return not isinstance(__value, Position) or (self.line, self.character) != (
+            __value.line,
+            __value.character,
+        )
+
+    def __gt__(self, __value: object) -> bool:
+        return isinstance(__value, Position) and (self.line, self.character) > (
+            __value.line,
+            __value.character,
+        )
+
+    def __ge__(self, __value: object) -> bool:
+        return isinstance(__value, Position) and (self.line, self.character) >= (
+            __value.line,
+            __value.character,
+        )
+
+    def __lt__(self, __value: object) -> bool:
+        return isinstance(__value, Position) and (self.line, self.character) < (
+            __value.line,
+            __value.character,
+        )
+
+    def __le__(self, __value: object) -> bool:
+        return isinstance(__value, Position) and (self.line, self.character) <= (
+            __value.line,
+            __value.character,
+        )
```

Comment:
Here some of them could be simplified by defining them in relation to other operators (e.g. lt is just not ge)